hadoop-ozone/dist/src/main/smoketest/s3/bucketlist.robot

```diff
@@ -30,4 +30,10 @@ ${BUCKET}             generated
 
 List buckets
     ${result} =         Execute AWSS3APICli     list-buckets | jq -r '.Buckets[].Name'
-                        Should contain          ${result}    ${BUCKET}
\ No newline at end of file
+                        Should contain          ${result}    ${BUCKET}
+
+List buckets with empty access id
+                        Execute                    aws configure set aws_access_key_id ''
+    ${result} =         Execute AWSS3APICli and checkrc         list-buckets    255
+                        Should contain            ${result}         The authorization header you provided is invalid
+    Run Keyword         Setup s3 tests
```

Comment:
There might be a issue here. If a new test case is added in bucketlist.robot. The test case would fail because of no access id.
If might confuse the user.
Maybe it's better to do a clean up here?

Reply:
I don't think we need to worry about this now.  New test case can be added above this one to avoid auth problem.

Reply:
Ok, I'll update the patch.